blog/authors.yml

```diff
@@ -4,32 +4,43 @@ BekahHW:
   image_url: https://github.com/BekahHW.png
   email: Bekah@opensauced.pizza
   socials:
-      x: bekahhw
-      linkedin: bekah-hawrot-weigel
-      github: bekahhw
-      site: https://www.bekahhw.com/
-  page: true 
+    x: bekahhw
+    linkedin: bekah-hawrot-weigel
+    github: bekahhw
+    site: https://www.bekahhw.com/
+  page: true
 
 nickytonline:
   name: Nick Taylor
   title: AI Engineer
   image_url: https://github.com/nickytonline.png
   email: nick@opensauced.pizza
   socials:
-      x: nickytonline
-      linkedin: nickytonline
-      github: nickytonline
-      site: https://www.nickyt.co/
-  page: true 
+    x: nickytonline
+    linkedin: nickytonline
+    github: nickytonline
+    site: https://www.nickyt.co/
+  page: true
 
 jpmcb:
   name: John McBride
   title: Sr Software Engineer | AI Engineer
   image_url: https://github.com/jpmcb.png
   email: john@opensauced.pizza
-  socials: 
+  socials:
     x: johncodezzz
     linkedin: jpmcb
     github: jpmcb
     site: https://johncodes.com/
-  page: true 
+  page: true
+
+AdiatiAyu:
+  name: Ayu Adiati
+  title: Tech Blogger | Documentation Team Lead | Monthly Challenge Team Lead
```

Comment:
Just a nit pick here: Can you add Virtual Coffee to the front of your team lead roles? You might want to have the entry be Virtual Coffee Documentation and Monthly Challenge Team Lead, but feel free to do it how you want it to appear.

Reply:
Thanks! I wanted to ask you how to format these and add Virtual Coffee to the roles. 😊